### ISSUE TITLE
A11y_SQLAzureDataStudio_ConnectToASQLServer_AdvancedProperties_ScreenReader: Focus order while navigating the scrollable content is improper due to which all content present is not read by screen reader. 

### ISSUE THREAD

**Issue by v-atkulk** [microsoft/azuredatastudio#9470]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11yMAS;#A11yTCS;#SQL Azure Data Studio;#DesktopApp;#MacOs;#Benchmark;#A11y_SQLAzureDataStudiobenchmark;#A11ySev2;#MAS1.3.1;#MAS2.4.3

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.15.0-insider
Commit: 3a8b74a
Date: 2020-02-11T05:49:41.181Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.3.0
Mac OS:10.15.3

**Additional Details:**
Screen reader: Voice Over
MAS reference: MAS1.3.1,MAS2.4.3
Mac OS:10.15.3

**Repro Steps:**
1.Launch Azure Data Studio.
2.Select the Connections icon (if not already selected) from left bar.
3.Hover over the 'Servers' header and select the 'New Connection' icon.
4.Navigate till Advanced button under Connection details & hit enter.
5.Launch Voice Over screen reader.
6.Check if all the data present on the screen is been read by screen reader.

**Actual:**
When we navigate through the information that is present on advanced properties screen, the content which is visible when we scroll the content is not been read by JAWS screen reader.

**Expected:**
All the content that is present on the screen should be read by the screen reader.

**User Impact:**
Users might receive less information as all the content present on the screen is not accessible for keyboard as well as screen reader users.

**Attachment link reference:**
[9470_A11y_SQLAzureDataStudio_ConnectToASQLServer_AdvancedProperties_ScreenReader Focus order while navigating the scrollable content is improper due to which all content present is not read by screen reader..zip](https://github.com/microsoft/azuredatastudio/files/4296572/9470_A11y_SQLAzureDataStudio_ConnectToASQLServer_AdvancedProperties_ScreenReader.Focus.order.while.navigating.the.scrollable.content.is.improper.due.to.which.all.content.present.is.not.read.by.screen.reader.zip)

**Comment by Charles-Gagnon** [comment]
Dupe of #9407

**Comment by aniketr99** [comment]
Verified the bug in below environment:
Version: 1.41.1 (Universal)
Commit: 33dce95b152227c4f10f6251ab30690fe9895c0c
Date: 2023-01-28T00:49:19.454Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Darwin arm64 22.3.0
issue is fixed hence adding #Closed; tag.

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-SQLADS-MAC-Feb2020;
